### PR TITLE
Re-introduce: update metatype name and desc for back-channel logout

### PR DESCRIPTION
**Pull request by jimmy1wu**
Reverts OpenLiberty/open-liberty#27300 for #27301

### CODE REVIEW

**Comment by jimmy1wu** [comment]
#build

File: dev/com.ibm.ws.security.openidconnect.server/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -74,6 +74,9 @@ sessionManaged.desc=Indicate by true or false whether session management is supp
 idTokenLifetime=ID token lifetime
 idTokenLifetime.desc=Time that ID token is valid (seconds). You can also specify a positive integer followed by a unit of time, which can be hours (h), minutes (m), or seconds (s). For example, specify 30 seconds as 30 or 30s. You can include multiple values in a single entry. For example, 1m30s is equivalent to 90 seconds.
 
+backchannelLogoutRequestTimeout=Back-channel logout request timeout
+backchannelLogoutRequestTimeout.desc=Specifies the amount of time after which a back-channel logout request to an OpenID Connect client times out.
```
Comment (dmuelle) [inline dev/com.ibm.ws.security.openidconnect.server/resources/OSGI-INF/l10n/metatype.properties]:
```suggestion
backchannelLogoutRequestTimeout.desc=Specifies the amount of time before a back-channel logout request to an OpenID Connect client times out.
```

**Review by dmuelle — DISMISSED** [review]
ID review- minor edit suggestion

**Comment by jimmy1wu** [comment]
#build

**Comment by jimmy1wu** [comment]
#build

**Review by ayoho — APPROVED** [review]

**Comment by jimmy1wu** [comment]
#libby

**Comment by jimmy1wu** [comment]
it has already been ID reviewed

**Pull request merged**